### ISSUE TITLE
ngb-timepicker - Manuel entering time values into input fields doesn't work on Ms-Edge 16-18

### ISSUE THREAD

**Issue by brabenetz** [ng-bootstrap/ng-bootstrap#3621]
Microsoft Edge 44.17763.831.0
Microsoft EdgeHTML 18.17763

This can be reproduced on every example of the ng-bootstrap example-page:
https://ng-bootstrap.github.io/#/components/timepicker/examples

I tested following ng-bootstrap versions
✓ 4.1.1
✓ 4.2.2
\- 5.0.0 ( compile errors: Couldn't resolve original symbol for ./datepicker/adapters/ngb-date-adapter ...)
✓ 5.1.0
✓ 5.1.5
X 5.2.0
X 5.2.1
X 5.2.3
\- 6.0.0 ( compile errors (e.g.:  error TS1086: An accessor cannot be declared in an ambient context.)

In other words:
It worked until Version 5.1.5 also in MS Edge 18.
Since version 5.2.0 it is broken.
6.0.0 I couldn't test because of breaking changes.

I also tested https://ng-bootstrap.github.io/#/components/timepicker/examples on multiple Ms Edge Browsers:
X EdgeHTML 18
X EdgeHTML 17
X EdgeHTML 16
✓ EdgeHTML 15

Since Ms-Edge version 16 (before Chromium) the current ng-bootstrap timepicker doesn't work anymore. It worked with EdgeHTML version 15.

**Comment by maxokorokov** [comment]
@brabenetz could you please clarify what exactly is not working / record a gif or video maybe? 

I can change time values without any issues in Edge 17

**Comment by brabenetz** [comment]
Just enter a new Value with the Keyboard:
![2020-02-26_ng-bootstrap_value-change](https://user-images.githubusercontent.com/1237858/75360989-dccdfa00-58b6-11ea-8e56-74367556d9e6.png)

**Comment by maxokorokov** [comment]
Oh this... let's say it is done "by design", because the model is updated only on blur... and it is not related to any specific browser

![Timepicker](https://user-images.githubusercontent.com/8074436/75429173-a8574e00-5949-11ea-96f9-08ae9a8be56b.gif)

Please reopen if necessary.

**Comment by brabenetz** [comment]
That's exactly what dos NOT work with MS-Edge 18!

Please reopen.

Thanks.

**Comment by brabenetz** [comment]
I reopened it: https://github.com/ng-bootstrap/ng-bootstrap/issues/3629

**Comment by maxokorokov** [comment]
> X EdgeHTML 18
> X EdgeHTML 17
> X EdgeHTML 16
> ✓ EdgeHTML 15
> 
> Since Ms-Edge version 16

@brabenetz you also told it was not working with Edge 17, but it actually is.

**Comment by maxokorokov** [comment]
Managed to reproduce in SauceLabs in Edge 18 on the demo site.

Looking at the history it might be related to this change → #3247 that added the `(input)` handler.
Similar issue in Angular repo → https://github.com/angular/angular/issues/34035

cc @fbasso

**Comment by benouat** [comment]
Closing for 2 reasons:

+ reverting #3247 performed seems to have fixed the issue (See previous @maxokorokov  comment)
+ I am not that sure we do support old version of Edge, now that the Chromium version has landed.

**Comment by petester42** [comment]
@benouat Just letting you know that this is still broken on Edge 18. What's the official stance on which browsers are supported?

**Comment by songokudbz** [comment]
Edge on MAC is working but Edge on Windows is not working...
